Vector/Assets/en.lproj/Vector.strings

```diff
@@ -95,6 +95,7 @@
 "room_creation_keep_private" = "Keep private";
 "room_creation_make_private" = "Make private";
 "room_creation_wait_for_creation" = "A room is already being created. Please wait.";
+"room_creation_invite_another_user" = "Search / invite by name, email, id";
```

Comment:
is "id" enough meaningful to end user?
We use "Matrix ID" in some other places.